backend/app.js

```diff
@@ -13,15 +13,17 @@ const ordersdb = require("./orderquery");
 app.use(cors());
 app.use(bodyParser.json());
 
-app.get("/inventory", async function (req, res) {
+app.get("/inventory", async function(req, res) {
   res.send(await getId());
 });
 
 async function getId() {
   let idList = [];
 
   const response = await fetch(
-    "https://api.bigcartel.com/v1/accounts/" + userId + "/products",
+    "https://api.bigcartel.com/v1/accounts/" +
+      userId +
+      "/products?page%5Blimit%5D=100",
```

Comment:
why add these parameters? if we have more data I don't think you have a fetch more method, I'd fetch all the data (because otherwise your sorting/filtering is technically incorrect since you could be missing data).

Reply:
I need that parameter to get all products

Reply:
the limit was 20 so I couldn't get all products